src/main/java/com/ubb/zenith/model/Song.java

```diff
@@ -30,4 +30,9 @@ public class Song {
     @JoinColumn(name = "mood_id", nullable = false)
     @JsonBackReference
     private Mood mood;
+
+    @ManyToOne
+    @JoinColumn(name = "playlist_id", nullable = false)
```

Comment:
Let's quit nullable = false this time
It'll cause an error. We try to avoid it till we find a solution